submissions/tests/test_team_api.py

```diff
@@ -237,6 +237,19 @@ def test_get_team_submission(self):
             TeamSubmissionSerializer(team_submission_model).data
         )
 
+    def test_get_team_submission_from_individual_submission(self):
+        """
+        Test that calling team_api.get_team_submission_from_individual_submission returns the expected team submission
+        """
+        team_submission_model = self._make_team_submission(create_submissions=True)
+        regular_submission_uuid = [team_submission_model.submissions.all()][0][0].uuid
```

Comment:
what's the second `[0]` there for? we have a list of lists of submissions? 
is it because we're putting the brackets around the all() and then having a list with one element which is the list of submissions? 

could we just do `list(team_submission_model.submissions.all())[0]`? or even better, `team_submission_model.submissions.first().uuid`?

Reply:
good catch. writing code Friday afternoon does interesting things to one's perspective.